lib/reports/matched_content_changes_report.rb

```diff
@@ -0,0 +1,43 @@
+class Reports::MatchedContentChangesReport
+  OUTPUT_ATTRIBUTES = {
+    created_at: :content_change,
+    base_path: :content_change,
+    change_note: :content_change,
+    document_type: :content_change,
+    publishing_app: :content_change,
+    priority: :content_change,
+    title: :subscriber_list,
+    subscription_count: :itself,
+  }.freeze
+
+  def call(start_time: nil, end_time: nil)
+    start_time = start_time ? Time.zone.parse(start_time) : 1.week.ago
+    end_time = end_time ? Time.zone.parse(end_time) : Time.zone.now
+
+    subscription_count_query = Subscription
+      .select("count(*)")
+      .where("subscriptions.created_at <= content_changes.created_at")
+      .where("subscriptions.ended_at is null or subscriptions.ended_at > content_changes.created_at")
+      .where("subscriptions.subscriber_list_id = subscriber_lists.id")
+      .where(frequency: "immediately")
+      .arel
+      .as("subscription_count")
+
+    query = MatchedContentChange
+      .where("content_changes.created_at": start_time..end_time)
+      .includes(:subscriber_list, :content_change)
+      .order(subscription_count: :desc)
+      .select("content_changes.*", "subscriber_lists.*")
+      .select(subscription_count_query)
+
+    CSV.generate do |csv|
+      csv << OUTPUT_ATTRIBUTES.keys
+
+      query.each do |row|
+        next if row.subscription_count.zero?
+
+        csv << OUTPUT_ATTRIBUTES.map { |sub_field, field| row.public_send(field).send(sub_field) }
```

Comment:
Thanks - you just missed the second `send`

Reply:
Oops again - I'm just naturally nosy, it seems. Fixed.